sdk/core/azure-core/src/main/java/com/azure/core/spatial/Geometry.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.spatial;
+
+/**
+ * An abstract representation of a geometry.
+ */
+public abstract class Geometry {
+    private final GeometryProperties properties;
+
+    Geometry(GeometryProperties properties) {
+        this.properties = properties;
+    }
+
+    /**
+     * Additional properties about this geometry.
+     *
+     * @return The additional properties associated with this geometry.
+     */
+    public GeometryProperties getProperties() {
+        return properties;
+    }
```

Comment:
Do you think, rather than have all Geometry subclasses having a getProperties() method, that Geometry itself should just contain the content of the GeometryProperties class? It seems quite long-winded to get a property at present, because you firstly need to getProperties.

Reply:
Based on the RFC geometry is a concept which is a set of positions and optional bounding box and additional properties, so having this be in a superclass makes sense based on the definition. I could see removing `GeometryProperties` for its direct types of bounding box and additional properties, flattening the structure by a level.